bundles/framework/layerlist/Flyout.js

```diff
@@ -110,7 +110,7 @@ Oskari.clazz.define('Oskari.mapframework.bundle.layerlist.Flyout',
          * Creates the UI for a fresh start
          */
         createUi: function () {
-            ReactDOM.render(<LayerViewTabs showOrganizations instance={this.instance} />, this.container);
+            ReactDOM.render(<LayerViewTabs showOrganizations instance={this.instance} locale={this.instance.getLocalization() } />, this.container);
```

Comment:
We have a LocaleProvider to handle this and getMsg() is the current version of locale-handling. But I heard this is some sort of compromise for now to reduce overlapping work with another developer so I'm fine with this.

Reply:
Add a comment about this in the code.

Reply:
Props will be changed in https://github.com/oskariorg/oskari-frontend/pull/1011 anyway.
We'll then continue working on replacing `locale` prop using `LocaleContext`.